roles/kubernetes-apps/policy_controller/calico/tasks/main.yml

```diff
@@ -1,14 +1,14 @@
----
 - set_fact:
     calico_cert_dir: "{{ canal_cert_dir }}"
   when: kube_network_plugin == 'canal'
-  tags: facts
+  tags: [policy-controller, canal]
```

Comment:
policy-controller tag is inherited from role. The tags here should be `facts, canal`